expression/builtin.go

```diff
@@ -961,3 +964,20 @@ func (b *baseBuiltinFunc) setDecimalAndFlenForTime(fsp int) {
 		b.tp.SetFlenUnderLimit(b.tp.GetFlen() + 1)
 	}
 }
+
+const emptyBaseBuiltinFunc = int64(unsafe.Sizeof(baseBuiltinFunc{}))
+
+// MemoryUsage return the memory usage of baseBuiltinFunc
+func (b *baseBuiltinFunc) MemoryUsage() (sum int64) {
+	if b == nil {
+		return
+	}
+
+	sum = emptyBaseBuiltinFunc + b.bufAllocator.MemoryUsage() +
+		b.tp.MemoryUsage() + int64(unsafe.Sizeof(*b.childrenVectorizedOnce)) +
```

Comment:
Can `childrenVectorizedOnce` and `childrenReversedOnce` be nil?

Reply:
I suggest to define a const number for `Once` like `const onceSize = int64(unsafe.Sizeof(Once{}))` instead of copying it here.